tests/ValidateSbrpTests/ValidateSbrp.Tests.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <Nullable>enable</Nullable>
+    <NoWarn>$(NoWarn);CS8602</NoWarn>
+    <NoWarn>$(NoWarn);CS8603</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NuGet.Packaging" Version="$(NuGetPackagingVersion)" />
+    <PackageReference Include="System.IO.Compression" Version="4.3.0" />
+    <PackageReference Include="System.IO.Compression.ZipFile" Version="4.3.0" />
```

Comment:
These old 4.3.0 packages shouldn't be referenced anymore these days. They will eventually be marked as deprecated. Curious, why did you bring them in? They shouldn't be needed anymore.

Reply:
I was experiencing compile errors for `ZipFile` (not being recognizined, needing to add the namespace, adding the namespace and it still giving the warnings -> added the package references).. adding the references fixed the issues. In hindsight, I think it's because I was using an old version of the sdk. Updating which sdk I was using fixed the issues I was having. Still a bit unsure of what fixed the issue I was having, but after creating a new container and reinstalling everything, I removed the package references and everything compiles as desired.